tamr_unify_client/client.py

```diff
@@ -68,20 +82,13 @@ def __init__(
         self._projects = ProjectCollection(self)
         self._datasets = DatasetCollection(self)
 
-        # logging
-        self.logger = None
-        # https://docs.python.org/3/howto/logging-cookbook.html#implementing-structured-logging
-
         if not self.base_path.startswith("/"):
             self.base_path = "/" + self.base_path
 
         if not self.base_path.endswith("/"):
             self.base_path = self.base_path + "/"
 
-        def default_log_entry(method, url, response):
-            return f"{method} {url} : {response.status_code}"
-
-        self.log_entry = None
+        self.response_message = _default_response_message
```

Comment:
`_default_response_message` is only used once; should it be in-lined?

Reply:
Also, the name `response message` is unclear - this looks quite specific to logging, so `log_message` would be good (or `format_for_log` or similar).
I don't see that it's relevant to put `default` in the name, since there's no way to provide a non-default.

Reply:
This was a poor-man's way to let users log the response body. I think you're right: we should just inline it and not make it accessible.